### PR TITLE
Add JDK13 to Jenkins Pipelines

### PR DESCRIPTION
**Pull request by AdamBrousseau**
- Update variable file and PBT-All

[skip ci]

Signed-off-by: Adam Brousseau <adam.brousseau88@gmail.com>

### CODE REVIEW

**Comment by AdamBrousseau** [comment]
Jenkins compile all jdk13

**Comment by AdamBrousseau** [comment]
jenkins compile xlinuxxl jdk13

**Comment by AdamBrousseau** [comment]
jenkins compile xlinux jdk13

**Comment by AdamBrousseau** [comment]
@jdekonin for review

[win](https://ci.eclipse.org/openj9/job/Build_JDK13_x86-64_windows_Personal/1/)
[plinux](https://ci.eclipse.org/openj9/job/Build_JDK13_ppc64le_linux_Personal/1/)
[zlinux](https://ci.eclipse.org/openj9/job/Build_JDK13_s390x_linux_Personal/1/)
[xlinux](https://ci.eclipse.org/openj9/job/Build_JDK13_x86-64_linux_Personal/2/)
[xlinuxxl](https://ci.eclipse.org/openj9/job/Build_JDK13_x86-64_linux_xl_Personal/1)
[osx](https://ci.eclipse.org/openj9/job/Build_JDK13_x86-64_mac_Personal/1/)
[aix](https://ci.eclipse.org/openj9/job/Build_JDK13_ppc64_aix_Personal/1/) (Failed as expected)

**Comment by jdekonin** [comment]
@pshipton for review/merge.

File: buildenv/jenkins/variables/defaults.yml [hunk 1]
```diff
@@ -62,6 +66,7 @@ jdk_image_dir:
   8: 'j2sdk-image'
   11: 'jdk'
   12: 'jdk'
+  13: 'jdk'
```
Comment (pshipton) [inline buildenv/jenkins/variables/defaults.yml]: Is there a way to make this easier in the future? i.e. we can specify `8:` and a `default:` for everything else? In many cases here and below the values don't change for jdk13.

File: buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All [hunk 2]
```diff
@@ -75,7 +75,7 @@
  *   OPENJDK<version>_SHA_<platform>: String - the last commit SHA
  */
 
-CURRENT_RELEASES = ['8', '11', '12', 'next']
+CURRENT_RELEASES = ['8', '11', '12', '13', 'next']
```
Comment (pshipton) [inline buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All]: I don't want to add 13 to the nightly build, I don't think we have the machine resources for all platforms. Plus the system tests will always fail until the problems are resolved.

Comment (AdamBrousseau) [reply buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All]: The nightly build (and any build) specifies what versions it wants to build so we'd have to add the boolean param to the nightly config in order to enable it.

File: buildenv/jenkins/variables/defaults.yml (hunk 1 above)
Comment (AdamBrousseau) [reply buildenv/jenkins/variables/defaults.yml]: I agree this is painful.
@dnakamura has an old PR #2016 which we should revisit.

**Pull request merged**